recipes/pescador/meta.yaml

```diff
@@ -0,0 +1,81 @@
+{% set version = "1.0.0" %}
+{% set hash_type = "md5" %}
+{% set hash_val = "5f3663fe66ce18f3f06be9ba5cbfbe1a" %}
+
+package:
+  name: pescador
+  version: {{ version }}
+
+source:
+  fn: pescador-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/p/pescador/pescador-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+#  patches:
+   # List any patch files here
+   # - fix.patch
+
+build:
+  number: 1
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+  # noarch_python: True
+  # preserve_egg_dir: True
+  # entry_points:
+    # Put any entry points (scripts to be generated automatically) here. The
+    # syntax is module:function.  For example
+    #
+    # - pescador = pescador:main
+    #
+    # Would create an entry point called pescador that calls pescador.main()
+
+
+  # If this is a new build for the same version, increment the build
+  # number. If you do not include this key, it defaults to 0.
+  # number: 1
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - joblib >=0.9
+    - six
+    - pyzmq
+    - numpy x.x
+
+  run:
+    - python
+    - joblib >=0.9
+    - six
+    - pyzmq
+    - numpy x.x
+
+test:
+  # Python imports
+  imports:
+    - pescador
+
+  # commands:
+    # You can put test commands to be run here.  Use this to test that the
+    # entry points work.
+
+
+  # You can also put a file called run_test.py in the recipe that will be run
+  # at test time.
+
+  # requires:
+    # Put any additional test requirements here.  For example
+    # - nose
+
+about:
+  home: http://github.com/pescadores/pescador
+  license: ISC
+  summary: 'Multiplex generators for incremental learning'
```

Comment:
The preference is to include a license_file in the conda package that gets built. In lieu of having the license file available when the conda package is built the request is usually   

(1) add a MANIFEST.in to the python package upstream and note that this was done so that when the conda package is built for that version, we can include the license with the conda package or  

(2) include the license in the recipe directory [like this](https://github.com/conda-forge/staged-recipes/pull/2689/commits/10c83cbb29211dedcab4269ae16a1e34543e3fdd#diff-9e023f20b425c0bdb974c3812be0b592R31).  

FWICT this is more of a suggestion than a **YOU MUST DO THIS** :smile:. Up to you on this. If you'd prefer to do neither of those things, then :+1: from me